app/views/providers/allocations/_request_status.html.erb

```diff
@@ -11,18 +11,28 @@
   </tr>
   </thead>
   <tbody class="govuk-table__body">
-  <% allocations.each do | allocation | %>
+  <% @training_providers.each do | training_provider | %>
```

Comment:
not a blocker but weird syntax. there's not normally whitespace around the block argument
```
<% @training_providers.each do |training_provider| %>
```

Reply:
good spot, will fix that